### PR TITLE
bluetooth: audio: fix code comments error

### PR DESCRIPTION
**Pull request by 0bijian0**
Added PB_Flag 0b10:The ISO_SDU_Fragment field contains a complete SDU.

### CODE REVIEW

File: subsys/bluetooth/host/iso.c [hunk 1]
```diff
@@ -592,7 +592,7 @@ void bt_iso_recv(struct bt_conn *iso, struct net_buf *buf, uint8_t flags)
 	BT_ISO_DATA_DBG("handle %u len %u flags 0x%02x pb 0x%02x ts 0x%02x",
 			iso->handle, buf->len, flags, pb, ts);
 
-	/* When the PB_Flag does not equal 0b00, the fields Time_Stamp,
+	/* When the PB_Flag does not equal 0b00 and 0b10, the fields Time_Stamp,
```
Comment (Thalley) [inline subsys/bluetooth/host/iso.c:595]:
```suggestion
	/* When the PB_Flag does not equal BT_ISO_START or BT_ISO_SINGLE, the fields Time_Stamp,
```

**Review by Thalley — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/host/iso.c (hunk 1 above)
Comment (0bijian0) [reply subsys/bluetooth/host/iso.c]: ok, I will make a new PR, tks

Comment (hermabe) [reply subsys/bluetooth/host/iso.c]: You dont have to create a new PR, you can just force-push a new version to the same branch and this PR will be updated.

Comment (0bijian0) [reply subsys/bluetooth/host/iso.c]: > You dont have to create a new PR, you can just force-push a new version to the same branch and this PR will be updated.

got it，tks

**Pull request closed without merging**